pytest.ini

```diff
@@ -1,3 +1,57 @@
 [pytest]
 addopts = -rs --ignore=setup.py --ignore=test_requirements.txt
 norecursedirs = .git build dist tmp* .eggs
+text_file_format = rst
```

Comment:
@Marie-Terrell I could not get away without any changes to to `pytest.ini`, so I'm asking for your review, too. However, not that the options are `doctest_*` and `test_file_format` and since we don't (yet) run doctests in CI, there should be no problem.

Reply:
I originally asked for @Marie-Terrell review, just because I touched the `pytest.ini` file. However, the options that I added are not used in CI, so I have removed her from the list of "required reviews". I'd still welcome any feedback if you have any, of course.